src/main/java/at/hannibal2/skyhanni/mixins/hooks/RenderLivingEntityHelper.kt

```diff
@@ -37,39 +33,43 @@ class RenderLivingEntityHelper {
         }
 
         fun <T : EntityLivingBase> setNoHurtTime(entity: T, condition: () -> Boolean) {
-            entityNoHurTime.add(entity)
             entityNoHurTimeCondition[entity] = condition
         }
 
-        fun <T : EntityLivingBase> setColorMultiplier(entity: T): Int {
+        fun <T : EntityLivingBase> setEntityColorWithNoHurtTime(entity: T, color: Int, condition: () -> Boolean) {
```

Comment:
I just tested some setEntityColor cases, im sure we can all replace with setEntityColorWithNoHurtTime without any downsides but actually benefits. e.g. the mining commission mob highligher loses their color while the hit damage gets shown.